modules/lambda-scan-ips/locals.tf

```diff
@@ -1,4 +1,4 @@
 locals {
   module_name   = basename(path.module)
-  rel_path_root = trimsuffix(path.module, "modules/${local.module_name}")
+  rel_path_root = trimsuffix(path.module, "modules/${local.module_name}") != "" ? trimsuffix(path.module, "modules/${local.module_name}") : "."
```

Comment:
Sigh. It's annoying that each local has to be updated. Would it be easier to move the logic to the main module and pass in the relative path as an input? 

Maybe not.

Reply:
Good question. But we can solve this is a previous PR and then rebase this one.